src/org/exoplatform/widget/MyActionBar.java

```diff
@@ -1,275 +1,276 @@
-                                              View.VISIBLE);
-    getActionBar().setVisibility(visibility);
-  }
-
-  @Override
-  public void setTitle(CharSequence title) {
-    getActionBar().setTitle(title);
-  }
-
-  @Override
-  public void setTitle(int titleId) {
-    setTitle(getString(titleId));
-  }
-
-  public ActionBar getActionBar() {
-    ensureLayout();
-    return mActionBarHost.getActionBar();
-  }
-
-  public ActionBarItem addActionBarItem(ActionBarItem item) {
-    return getActionBar().addItem(item);
-  }
-
-  public ActionBarItem addActionBarItem() {
-
-    ActionBarItem item = getActionBar().newActionBarItem(NormalActionBarItem.class);
-
-    return getActionBar().addItem(item);
-
-  }
-
-  public ActionBarItem addActionBarItem(int itemId) {
-
-    @SuppressWarnings("deprecation")
-    final Drawable d = new ActionBarDrawable(this.getResources(), itemId);
-
-    ActionBarItem item = getActionBar().newActionBarItem(NormalActionBarItem.class).setDrawable(d);
-
-    return getActionBar().addItem(item);
-
-  }
-
-  public ActionBarItem addActionBarItem(ActionBarItem item, int itemId) {
-    return getActionBar().addItem(item, itemId);
-  }
-
-  public ActionBarItem addActionBarItem(ActionBarItem.Type actionBarItemType) {
-    return getActionBar().addItem(actionBarItemType);
-  }
-
-  public ActionBarItem addActionBarItem(ActionBarItem.Type actionBarItemType, int itemId) {
-
-    return getActionBar().addItem(actionBarItemType, itemId);
-  }
-  
-
-  public FrameLayout getContentView() {
-    ensureLayout();
-    return mActionBarHost.getContentView();
-  }
-
-  public void setActionBarContentView(int resID) {
-    LayoutInflater.from(this).inflate(resID, getContentView());
-    Drawable bg = getActionBar().getBackground();
-    if (bg != null && (bg instanceof BitmapDrawable)) {
-      BitmapDrawable bmDrawable = (BitmapDrawable) bg;
-      bmDrawable.setTileModeX(TileMode.REPEAT);
-      getActionBar().setBackgroundDrawable(bmDrawable);
-    }
-  }
-
-  public void setActionBarContentView(View view, LayoutParams params) {
-    getContentView().addView(view, params);
-  }
-
-  public void setActionBarContentView(View view) {
-    getContentView().addView(view);
-  }
-
-  public boolean onHandleActionBarItemClick(ActionBarItem item, int position) {
-
-    return true;
-  }
-
-  private OnActionBarListener mActionBarListener = new OnActionBarListener() {
-                                                   public void onActionBarItemClicked(int position) {
-
-                                                     onHandleActionBarItemClick(getActionBar().getItem(position),
-                                                                                position);
-
-                                                   }
-                                                 };
-
-  public GDApplication getGDApplication() {
-    return (GDApplication) getApplication();
-  }
-
-}
+// // TODO delete
```

Comment:
For the Toolbar, it is a bit late to add it at this step, but I opened an issue to study and maybe do it later (MOB-1876).